kolibri/core/assets/src/vue/session-nav-widget/index.vue

```diff
@@ -79,10 +79,7 @@
         return this.fullname;
       },
       userkind() {
-        if (this.deviceOwner) {
-          return '';
-        }
-        return this.kind;
+        return this.kind[0];
```

Comment:
Still iffy on whether we should make this more robust (searching for the highest one), but if the API always returns it this way, I guess we don't need to do any further checks.